networktest.go

```diff
@@ -234,7 +249,14 @@ func (l *lightningNode) cleanup() error {
 
 // stop attempts to stop the active lnd process.
 func (l *lightningNode) stop() error {
-	if l.cmd == nil || l.cmd.Process == nil {
+
+	// We should skip node stop in case:
+	// - start of the node wasn't initiated
+	// - process wasn't spawned
+	// - process already finished
+	if 	l.cmd == nil ||
```

Comment:
I still need to get up a document detailing contribution guidelines, but this file should be run through `go-fmt`.